store/localstore/local_region.go

```diff
@@ -403,23 +411,32 @@ func (rs *localRegion) getRowByHandle(ctx *selectContext, handle int64) (*tipb.R
 	return row, nil
 }
 
-func (rs *localRegion) evalWhereForRow(ctx *selectContext, h int64) (bool, error) {
+// TODO: find a better way to decode row data.
+func (rs *localRegion) getRowData(value []byte, colTps map[int64]*types.FieldType) (map[int64][]byte, error) {
+	values, err := tablecodec.DecodeRow(value, colTps)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	ret := make(map[int64][]byte, len(colTps))
+	for id, value := range values {
```

Comment:
It's unnecessary to encode here. Because following operation, like where evaluation or aggregation, also need datum.